app/views/camps/show.html.erb

```diff
@@ -371,6 +371,19 @@
                     <div class="panel-heading"><h2 class="header-sub-heading"><%=t :creation_info_headline %></h2></div>
                 <div class="panel-body">
 
+                <% if @camp.active? %>
```

Comment:
Also - can you make it into a feature that can be turned on / off (in sake of borderland and others).
(add it as a setting to firestarter_settings.xml file with ability to update it using an ENV var and default - turn it off.